src/factory.ts

```diff
@@ -564,6 +575,20 @@ async function flashArcSlot(
             `userdata`
         )
         lastUserdataEntry = userdataEntry
+
+        if (ablEntry) {
+            try {
+                console.log(`flashing abl`);
+                await flashEntryBlob(
+                    device,
+                    ablEntry,
+                    onProgress,
+                    `abl`
+                )
+            } catch (e) {
+                // ignore
```

Comment:
I would still add an error, if we add the abl and it fails during flashing I think it's relevant to know.